src/variable.cpp

```diff
@@ -146,8 +146,8 @@ std::string interfaceTypeToString(Variable::INTERFACE_TYPES interfaceType)
     std::string str;
     switch (interfaceType) {
     case Variable::INTERFACE_TYPES::INTERFACE_TYPE_NONE: {
-        str = "none";
-        break;
+        /* Should not ask for the string version of this.
+        With the current codebase there is no way to trigger this case. */
```

Comment:
I don't like this. Why couldn't I ask for the string version of the none type? In fact, if I was to ask for such a string, I would currently get ```private``` which is clearly wrong. Personally, I would revert that change (and have something similar to what I have done in for PREFIX_UNIT in commit https://github.com/cellml/libcellml/commit/022e30f4d59ed177372ee5fc604f647482ed5796).